### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by AnkitaSikdar005**
## What does this PR do?
Add resource(s) 

## For resources
Added Cloud Computing Courses

### Why is this valuable (or not)?
Yes, it is valuable.
### How do we know it's really free?
It is available on YouTube.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by AnkitaSikdar005** [comment]
@sohansai Please merge this pull request.

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Comment by AnkitaSikdar005** [comment]
@eshellman Made the changes as told by you.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -381,6 +381,7 @@
 * [A Practical Introduction to Cloud Computing](https://www.udemy.com/course/introduction-cloud-computing/) - Neil Anderson
 * [Cloud Computing and Development](https://uaceit.com/courses/cloud-computing-and-development) - Ashwin Kumar Ramaswamy (UAceIt) (email address *required*)
 * [Cloud Computing Full Course In 11 Hours \| Cloud Computing Tutorial For Beginners](https://www.youtube.com/watch?v=2LaAJq1lB1Q) - Edureka!
+* [Cloud Computing intro and applications](https://www.youtube.com/watch?v=G9LSFoBA-Bo&list=PLROvODCYkEM8xUk8R2T79KbZ7YfCXMbZS&index=1) - OU Education
```
Comment (sohansai) [inline courses/free-courses-en.md:384]: We prefer 'playlist' links for YouTube contributions, and refrain from creating new titles. Use the title provided by the source.
```suggestion
* [Cloud Computing (CC)](https://www.youtube.com/playlist?list=PLROvODCYkEM8xUk8R2T79KbZ7YfCXMbZS) - OU Education
```

**Comment by AnkitaSikdar005** [comment]
@eshellman Made the changes as told by you.

**Pull request merged**